app/webrtc/client/tabBar.tsx

```diff
@@ -0,0 +1,22 @@
+import { useMemo } from 'react';
+
+import { useSetting } from '../../../client/contexts/SettingsContext';
+import { addAction } from '../../../client/views/room/lib/Toolbox';
+import { useTranslation } from '../../../client/contexts/TranslationContext';
+
+addAction('webRTCVideo', () => {
+	const t = useTranslation();
+	const enabled = useSetting('WebRTC_Enabled') && (useSetting('Livechat_call_provider') === t('WebRTC'));
+
+	const handleClick = () => { window.open('/', '_blank'); };
```

Comment:
![image](https://user-images.githubusercontent.com/34130764/124385916-79977780-dcf5-11eb-8b15-b7a72be36539.png)
New warning getting introduced bcs of this code. Please resolve them